manager/manager.go

```diff
@@ -237,30 +220,30 @@ func (m *Manager) AddEmitterFunc(emitter func(result transaction.Result)) {
 	m.AddEmitter(EmitterFunc(emitter))
 }
 
-// GetTransaction fetches a existing transaction from the manager
-func (m *Manager) GetTransaction(name string) (tx transaction.Transaction, err error) {
-	config, err := m.transactions.Get(name)
-	if err != nil {
-		return nil, fmt.Errorf("manager: %w", err)
-	}
-
-	return m.New(config)
-
+// Get a existing transaction from the manager
```

Comment:
Specify the `Get` is by name as specified in the `Delete` function. Just for consistency